tests/integrations/test_db_integrations_sql.py

```diff
@@ -165,6 +168,7 @@ def test_db_statement_query(self):
     @missing_feature(library="nodejs", reason="not implemented yet")
     @missing_feature(library="python", reason="not implemented yet")
     @missing_feature(library="python_otel", reason="Open Telemetry doesn't send this span for  python")
+    @missing_feature(library="nodejs_otel", reason="Open Telemetry doesn't send this span for  python")
```

Comment:
```suggestion
    @missing_feature(library="nodejs_otel", reason="Open Telemetry doesn't send this span for nodejs")
```

And also, do we know why ot does not send them ?